HeliPort/Appearance/StatusMenu.swift

```diff
@@ -43,6 +46,11 @@ final class StatusMenu: NSMenu, NSMenuDelegate {
 
             statusItem.title = NSLocalizedString(status.description)
 
+            if status != ITL80211_S_RUN && previousStatus == ITL80211_S_RUN && !isDisconnecting {
+                disassociateSSID(disconnectItem)
+                NetworkManager.scanSavedNetworks()
```

Comment:
Currently we're only performing auto-join when HeliPort is initially launched. Users might want to temporarily disconnect while keeping WiFi enabled. Apple’s approach supports this (likely to maintain BSSID-based location services). Although there’s minimal justification for us to replicate this behavior aside from consistency with Apple (aligns with user habits and expectations) and allowing users to view the network list while disconnected. See #133.